### DISCUSSION TITLE
Hello everyone!

### DISCUSSION THREAD

**Discussion by ddazal** [datasketch/frontend-coding-test#1] [Show and tell]
## 👋 Welcome!
Please drop the link of your GitHub repository with the solution to the requirements listed in the README.md file.

Best of luck to all of you!

**Comment by jtnovellis** [comment]
Hi! This is my solution to the requirements of the challenge. [This is the link to my repository](https://github.com/jtnovellis/frontend-coding-test)

**Comment by pipeleon** [comment]
Hello! Here's my [solution](https://github.com/pipeleon/frontend-coding-test)

**Comment by Koolxtreme** [comment]
Hello world!, I'll leave the [repository](https://github.com/Koolxtreme/frontend-coding-test) with the solution as specified, thanks a lot for reviewing, I would like to receive feedback if necesary to improve.

Sincerely [@koolxtreme](https://github.com/Koolxtreme)

**Comment by xKeCo** [comment]
Hi, This is my solution [Repository link](https://github.com/xKeCo/frontend-coding-test).

**Comment by ivancho2802** [comment]
Hi! This is my solution to the requirements of the challenge. [This is the link to my repository](https://github.com/ivancho2802/frontend-coding-test)

**Comment by JtabaresH** [comment]
Hello, David. I've attached my development test solution to this email. [This is the link to my repository](https://github.com/JtabaresH/frontend-coding-test)
Thank you in advance, and have a wonderful day!

**Comment by Juanse036** [comment]
Hello, Here it is my test solution, thank you [Repository Link](https://github.com/Juanse036/frontend-coding-test)

**Comment by gusmol37** [comment]
Hello David. Good afternoon. This is my solution to frontend-coding-test. In the following link you will find my repository: [Repository link](https://github.com/gusmol37/front-end-test.git) I appreciate your attention and your time.

**Comment by lucasvelez78** [comment]
Hello. This is my solution: [https://github.com/lucasvelez78/frontend-coding-test](url)

**Comment by mishino123** [comment]
Hello!!   good  night. send my solutions. [aqui muchas gracias]( https://github.com/mishino123/frontend-coding-test.git)

**Comment by jeniferSamper** [comment]
Hello, I send [my solution ](https://github.com/jeniferSamper/frontend-coding-test). thanks

**Comment by Angela2201** [comment]
Hello, good night, I send my [solution](https://github.com/Angela2201/frontend-coding-test). Thanks!

**Comment by Ka0o0zz** [comment]
Hello, good night, I send my [solution](https://github.com/Ka0o0zz/frontend-coding-test). !

**Comment by kelly1801** [comment]
Hello, Good night, here is my [solution](https://github.com/kelly1801/frontend-coding-test),
thanks.

**Comment by EstefaniAzocar** [comment]
Hello!! good night. send [my solutions.](https://github.com/EstefaniAzocar/frontend-coding-test)

**Comment by Davidalbort** [comment]
Hello good evening!, send [Link resolve challange](https://github.com/Davidalbort/frontend-coding-test.git). Good nigth

**Comment by afbayonac** [comment]
Hi,  [here](https://github.com/afbayonac/frontend-coding-test) is my solution

**Comment by ghost** [comment]
Hola :)
Esta es mi solución: [repo](https://github.com/daig0r/frontend-coding-test). Gracias!

**Comment by ddazal** [comment]
Dear all, thank you for your responses to the coding test. We will be reviewing your solutions and getting in touch during the week.
